src/args.rs

```diff
@@ -46,6 +46,14 @@ pub struct Arguments {
     #[arg(long = "nocapture", help = "No-op (libtest-mimic always runs in no-capture mode)")]
     pub nocapture: bool,
 
+    /// No-op, ignored (libtest-mimic doesn't support this)
+    #[arg(long = "show-output")]
+    pub show_output: bool,
+
+    /// Enable nightly-only flags
```

Comment:
I think this should say something like this:

```suggestion
    /// No-op, ignored. Flag only exists for CLI compatibility with libtest.
```

The libtest description doesn't really make sense in the context of libtest-mimic.